gatsby-browser.js

```diff
@@ -55,4 +55,27 @@ const onClientEntry = () => {
   window.ReactDOM = require('react-dom');
 };
 
-export { wrapPageElement, onInitialClientRender, onClientEntry };
+const shouldUpdateScroll = ({ routerProps: { location } }) => {
+  const PAGE_OFFSET_HEIGHT = 575;
```

Comment:
I think we may want to target the bottom of the banner for this, the current height slightly cuts off the top of the search bar.